apps/sozip.cpp

```diff
@@ -701,11 +646,11 @@ MAIN_START(nArgc, papszArgv)
         return 1;
 
     uint64_t nCurSize = 0;
-    for (int i = 0; i < aosFiles.size(); ++i)
+    for (int i = 0; i < static_cast<int>(aosFiles.size()); ++i)
```

Comment:
is there any reason for not using size_t here like you did in https://github.com/OSGeo/gdal/pull/9445/files#diff-256b9e29576d48c38ec9a84143ec9c1dd2f51b1e8eccc73f69807aff255899f3R627 ?

Reply:
changed to size_t